department-of-veterans-affairs/caseflow
# Justin/APPEALS-13222

Resolves [APPEALS-13222](https://vajira.max.gov/browse/APPEALS-13222)

### Description
Adding CAVC Dashboard button to the CAVC Details page on a CAVC Appeal. 

### Acceptance Criteria
- [ ] If a CAVC Remand for an appeal exists and the following conditions have been met in the CAVC Remand then in the CAVC Remand section on the Case Details page a "CAVC Dashboard" button is included
**If the user is in one of these organizations**
- [ ] OCC
- [ ] OAI

### Testing Plan
1. Log in as an OCC user
2. Find or create a CAVC Appeal
3. Go to the appeal details page
4. Scroll down to the CAVC Details section
5. Verify that button is present on the right-hand side
6. Verify that the button takes you to the CAVC Dashboard frame
7. Repeat process as an OAI user

### User Facing Changes

 BEFORE|AFTER
 ---|---

### Code Documentation Updates
